Patch: 71_date_overflows.patch
Author: Lionel Elie Mamane <lionel@mamane.lu>
React sensibly on integer overflow in date handling
Index: mailman-2.1.10/Mailman/Handlers/Scrubber.py
===================================================================
--- mailman-2.1.10.orig/Mailman/Handlers/Scrubber.py	2008-06-15 18:49:54.000000000 +0200
+++ mailman-2.1.10/Mailman/Handlers/Scrubber.py	2008-06-15 18:49:54.000000000 +0200
@@ -143,7 +143,7 @@
                      }.get(parts[3], 0)
             day = int(parts[4])
             year = int(parts[6])
-        except (IndexError, ValueError):
+        except (OverflowError, IndexError, ValueError):
             # Best we can do I think
             month = day = year = 0
         datedir = '%04d%02d%02d' % (year, month, day)
